src/server/controllers/users.controller.js

```diff
@@ -2,28 +2,6 @@
 
 const user = require('mongoose').model('user');
 
-let mockData = {
-  userId: 'gildong',
-  email: 'abcds@fsdss.com',
-  name: '홍길동',
-  age: 25,
-  field: 'SW Engineer',
-  region: 'Seoul',
-  skills: 'JAVA',
-};
-
-// Add a mock data into response.
-export function insertMockData(req, res, next) {
```

Comment:
This insertMockData method was for initial back-end test, so it's not needed for now. Maybe sign up flow will replace this!